### ISSUE TITLE
Repackaging of the cli?

### ISSUE THREAD

**Issue by DrAnaximandre** [tboquet/python-alp#93]
**OS:** Ubuntu 16.04 LTS
**Setup:** brand new machine

**Problem:** When installing on a brand new system, we actually require a lot more than just pip and docker.
Launching `alp --help` returns an error  for it tries to launch alp (with `appcom`, `core` and so on), thus it eventually crashes when trying to import `numpy` or whatever library used in the backend.

**Discussion** : What about separating the cli and the library ? If not, maybe we can try not to import the whole package when launching the services via the cli ?

**Comment by tboquet** [comment]
Right, this part should be improved. We should find a way of removing the dependencies when using the CLI.
